app/src/main/java/org/simple/clinic/teleconsultlog/teleconsultrecord/TeleconsultRecord.kt

```diff
@@ -0,0 +1,46 @@
+package org.simple.clinic.teleconsultlog.teleconsultrecord
+
+import androidx.room.Dao
+import androidx.room.Embedded
+import androidx.room.Entity
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy.REPLACE
+import androidx.room.PrimaryKey
+import androidx.room.Query
+import androidx.room.Relation
+import org.simple.clinic.drugs.PrescribedDrug
+import java.util.UUID
+
+@Entity
+data class TeleconsultRecord(
```

Comment:
This is quite a complex model. Can we write a small integration test that constructs a complete object of this type and saves it into the database and then reads it back to verify that it was done correctly?